### ISSUE TITLE
Installer seems to add `-e` to ~/.profile

### ISSUE THREAD

**Issue by cognivore** [numtide/nix-unstable-installer#27]
**Describe the bug**

After installing, whenever I drop into `bash`, it would greet me with `-e: command not found`.
It is due to the fact that nix-unstable-installer seems to add a line consisting of just `-e` into `~/.profile`.

**To Reproduce**

1. Install Ubuntu in WSL2
2. Run `sh <(curl -L https://github.com/numtide/nix-unstable-installer/releases/download/nix-2.5pre20211026_5667822/install)`
3. Exit shell
4. Enter shell by running `wsl.exe` or `ssh localhost`
5. Observe `-e: command not found`

**Expected behavior**

There is no extra output.

**System information**

OS: Ubuntu 20.04 focal(on the Windows Subsystem for Linux)
Kernel: x86_64 Linux 4.19.128-microsoft-standard

**Additional context**

`~/.profile` before installation:

```
λ cat ~/.profile
# ~/.profile: executed by the command interpreter for login shells.
# This file is not read by bash(1), if ~/.bash_profile or ~/.bash_login
# exists.
# see /usr/share/doc/bash/examples/startup-files for examples.
# the files are located in the bash-doc package.

# the default umask is set in /etc/profile; for setting the umask
# for ssh logins, install and configure the libpam-umask package.
#umask 022

# if running bash
if [ -n "$BASH_VERSION" ]; then
    # include .bashrc if it exists
    if [ -f "$HOME/.bashrc" ]; then
        . "$HOME/.bashrc"
    fi
fi

# set PATH so it includes user's private bin if it exists
if [ -d "$HOME/bin" ] ; then
    PATH="$HOME/bin:$PATH"
fi

# set PATH so it includes user's private bin if it exists
if [ -d "$HOME/.local/bin" ] ; then
    PATH="$HOME/.local/bin:$PATH"
fi
. "$HOME/.cargo/env"
```

**Comment by lilyinstarlight** [comment]
Hi there! This repo copies and serves passing unstable builds of Nix, including the original install script mostly unmodified (except to point it to the tarballs here instead of non-existent ones on release.nixos.org)

It looks like this bug has been reported and fixed upstream (NixOS/nix#5458), but a fixed version hasn't been able to make it to this repo yet because the macOS builds have been failing for a bit over a month now (see the [Hydra jobset](https://hydra.nixos.org/jobset/nix/master/evals)) and this repo only automatically updates to passing builds (also related - NixOS/nix#5687)

You should just be able to remove the `-e \n` from the beginning of that line (or similar so the line starts with `if`) in your `~/.profile` and be good. Does that resolve the issue for you?

**Comment by zimbatm** [comment]
^ what lily said. Issues like that are for upstream.
